rules/S6875/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "SequencedCollection method should be used to add or remove first or last element",
```

There are several methods:
```suggestion
  "title": "SequencedCollection methods should be used to add or remove first or last element",
```